web/src/repo/tree/TreePage.tsx

```diff
@@ -51,7 +51,11 @@ const TreeEntry: React.FunctionComponent<{
 }> = ({ isDir, name, parentPath, url }) => {
     const filePath = parentPath ? parentPath + '/' + name : name
     return (
-        <Link to={url} className={`tree-entry ${isDir ? 'font-weight-bold' : ''}`} title={filePath}>
+        <Link
+            to={url}
+            className={`tree-entry ${isDir ? 'font-weight-bold' : ''} e2e-tree-entry__${isDir ? 'directory' : 'file'}`}
```

Comment:
Our e2e selectors generally don't follow BEM (if they did, this would also be `--` not `__`, but that distinction doesn't really make sense for e2e selectors so we should just keep them flat)
```suggestion
            className={`tree-entry ${isDir ? 'font-weight-bold' : ''} e2e-tree-entry-${isDir ? 'directory' : 'file'}`}
```

Reply:
Renamed.